cosmoz-sse.js

```diff
@@ -0,0 +1,127 @@
+// @license Copyright (C) 2019 Neovici AB - Apache 2 License
+const SSEHandlers = {
+	text: data => data,
+	json: data => JSON.parse(data)
```

Comment:
Can we guarantee this will work or should we wrap JSON.parse in a try/catch?

Reply:
This should work as long as the server outputs proper JSON. If something is not configured right, the native error that is thrown should provide enough context for the developer to fix it. Wrapping the parse in a try/catch would only be useful to add additional context to the error.

Reply:
Well i just want to make sure that the component doesn't break completely from a corrupt message, just "console.error" and return null/undefined

Reply:
I can add the try/catch, but I don't think it's necessary. Considering that `JSON.parse` is called in an event handler, only that invocation will throw, so it won't break the component. Even in case of an error, the component will continue to listen and handle events from the server.